File: MySQL-NET-REST-API/DocumentManagement.Data/Entities/DocumentMetaData.cs
using DocumentManagement.Data.Entities;
using Microsoft.EntityFrameworkCore.Metadata.Internal;
using System;
using System.ComponentModel.DataAnnotations.Schema;

namespace DocumentManagement.Data;

public class DocumentMetaData : BaseEntity
{
    public Guid Id { get; set; }
    public Guid DocumentId { get; set; }
    public Document Document { get; set; }
    public Guid DocumentMetaTagId { get; set; }
    [ForeignKey("DocumentMetaTagId")]
    public DocumentMetaTag DocumentMetaTag { get; set; }
    public string Metatag { get; set; }
    public DateTime? MetaTagDate { get; set; }

}

File: AngularFrontend/src/app/core/domain-classes/document-meta-data.ts
import { MetaTagType } from "./meta-tag-type.enum";

export interface DocumentMetaData {
  id?: string;
  documentId?: string;
  documentMetaTagId?: string;
  metatag?: string;
  metaTagType?: MetaTagType;
  metaTagDate?: string;
}
